contractcourt/channel_arbitrator.go

```diff
@@ -321,6 +322,8 @@ func (h HtlcSetKey) String() string {
 // broadcasting to ensure that we avoid any possibility of race conditions, and
 // sweep the output(s) without contest.
 type ChannelArbitrator struct {
+	chainio.BeatConsumer
```

Comment:
I think this needs to come after the atomic variables, otherwise we might run into alignment issues in 32-bit systems.